sympy/benchmarks/bench_symbench.py

```diff
@@ -54,13 +54,13 @@ def uniq(x):
 
 def bench_R6():
     "sum(simplify((x+sin(i))/x+(x-sin(i))/x) for i in range(100))"
```

Comment:
Do you know what the point of having these strings here is?

Reply:
I'm not sure but I guess they show in the output when running the benchmarks.

Maybe sympy/benchmarks should be entirely removed now that we have the sympy_benchmarks repo.